app/views/components/_banner.html.erb

```diff
@@ -14,13 +14,13 @@
 <section class="app-c-banner<% if aside %> app-c-banner--grid<% end %>" aria-label="Notice">
   <% if aside %>
     <div class="grid-row">
+      <div class="app-c-banner__main">
+        <%= content_block %>
+      </div>
       <div class="column-third">
-        <div class="app-c-banner__aside">
+        <aside class="app-c-banner__aside">
```

Comment:
I'm not sure this should be an aside, despite the variable name. The content we're putting in there is not tangential, it's important and directly related to the main content. (https://developer.mozilla.org/en/docs/Web/HTML/Element/aside)

Reply:
Might be worth changing the class

Reply:
And the input variable name.